src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -814,24 +814,13 @@
   <data name="ColorScheme_RenameErrorTip.Title" xml:space="preserve">
     <value>This color scheme name is already in use.</value>
   </data>
-  <data name="SettingContainer_OverrideIntro" xml:space="preserve">
-    <value>This overrides the value from {}.</value>
-    <comment>"{}" is the object being overridden. Generally, it will be the name of another profile.</comment>
-  </data>
-  <data name="SettingContainer_ResetButtonHelpText" xml:space="preserve">
-    <value>Reset</value>
-  </data>
-  <data name="SettingContainer_OverrideTargetGenerator" xml:space="preserve">
-    <value>an autogenerated profile</value>
-  </data>
   <data name="Globals_FocusFollowMouse.Header" xml:space="preserve">
     <value>Enable Focus Follow Mouse mode</value>
   </data>
   <data name="Globals_FocusFollowMouse.HelpText" xml:space="preserve">
     <value>When checked, the terminal will the focus pane on mouse hover.</value>
   </data>
-  <data name="SettingContainer_OverrideTarget" xml:space="preserve">
-    <value>a lower layer</value>
-    <comment>This is the object of "SettingContainer_OverrideIntro".</comment>
+  <data name="SettingContainer_OverrideMessageBaseLayer" xml:space="preserve">
+    <value>Reset to base layer value</value>

```

Comment:
if we're only gonna do base layer, should we still add all of the source tracking? It seems our new needs are satisfied by the existence of Has/Clear in the model.

Reply:
We still need source tracking for a few reasons:
1. need to specifically detect when we're overriding "base layer" as opposed to "in-box" or "generated" profile object
2. we'll probably use some of this for telemetry
3. in preparation for profile inheritance, we'll still need to distinguish a profile that exists from one that isn't exposed to the user